e2e/fixtures/ssg-performance/src/pages/[path].tsx

```diff
@@ -1,8 +1,8 @@
 import { Path } from '../Path.js';
 
-export default async function Test() {
+export default async function Test({ path }: { path: string }) {
   await new Promise((resolve) => setTimeout(resolve, 1000));
-  return <Path />;
+  return <Path path={`/${path}`} />;
```

Comment:
@dai-shi should the `path` from props include the leading `/`? I think it should. This would make it consistent with `window.location.pathname`

maybe we tackle that in a new PR though if you agree?

Reply:
That's weird. When we say `path` or `pathname` it should start with `'/'`. How come this is happening? If it's a bug already, we would like to fix it before this PR.

Reply:
ok! I'll get a fix for this up then 👍

Reply:
this is fixed now... It was actually relying on `[path].tsx` as the filename and it got `path` from the mapping. So I changed the name of the file to `[slug].tsx` and passed `path` into the page props where I had previously missed it.

Follow up question: should we ban `[path].tsx` to avoid name collision for the props?

Reply:
> Follow up question: should we ban `[path].tsx` to avoid name collision for the props?

Yeah, using `createPages` allows it, but `fs-router` should warn or error.

Reply:
added an error for this scenario in fs-router